### ISSUE TITLE
Divider color and size

### ISSUE THREAD

**Issue by soareseneves** [WindSekirun/ActionSheet#3]
Hello,

May you please implement a way to customize the divider height and color? It would be cool to be able to increase it to 2px instead of 1 and the color to transparent, that would make it look like iOS 12 ActionSheet!

Thanks!

**Comment by WindSekirun** [comment]
I don't have any iOS device, can you give screenshot that can compare?

**Comment by soareseneves** [comment]
Yes, here it is, but instead of 2px like I said, 1px is fine, changing the color to transparent will do the trick!

Here is the iOS image:

![iOS](https://user-images.githubusercontent.com/16960561/66048959-e407fc00-e500-11e9-9b48-87081da4c923.png)

Here is the android image with the transparent divider color!

![android](https://user-images.githubusercontent.com/16960561/66048978-ee29fa80-e500-11e9-8f0c-dee933772c0b.jpg)
